api/src/domain/tax-filings/taxIdHelper.ts

```diff
@@ -0,0 +1,34 @@
+import { TaxFiling } from "@shared/taxFiling";
+import { TaxFilingResults } from "src/domain/types";
+
+export const reformatTaxId = (id: string) => id.trim().replaceAll(" ", "").replaceAll("/", "_").toLowerCase();
```

Comment:
would be nice if this was named in a way that we know what "reformat" means. Reformat to what? For what?